tests/kuttl/tests/ovn_nicmappings/01-assert.yaml

```diff
@@ -0,0 +1,344 @@
+#
+# Check for:
```

Comment:
can symlink this test file to the the ones in common directory as done in other tests.

Reply:
I'm still working on this. I don't know what is happening with the changes I made, kuttl tests are not passing.